server/index.js

```diff
@@ -19,11 +19,14 @@ setup(app, {
   publicPath: '/',
 });
 
-// get the intended port number, use port 3000 if not provided
+// get the intended host and port number, use localhost and port 3000 if not provided
+const host = argv.host || process.env.HOST || 'localhost';
 const port = argv.port || process.env.PORT || 3000;
 
+console.log(argv);
+
 // Start your app.
-app.listen(port, (err) => {
+app.listen(port, host, (err) => {
```

Comment:
This is actually internal express thing (i.e app.listen() is an express call). It has host parameter after post. That's why I didn't change logger to have host before port.